dev/firely/docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+  firely:
+    image: firely/server
+    ports:
+      - "8088:4080"
+    volumes:
+      - ./license/firelyserver-license.json:/app/firelyserver-license.json
+      - ./settings/appsettings.instance.json:/app/appsettings.instance.json
+      - ./settings/logsettings.instance.json:/app/logsettings.instance.json
+      - ./data:/app/data
```

Comment:
If I understand correctly, Firely is using this for the SQLite DB files. I was able to use MSSQL for everything (see comment RE:Administration), so this shouldn't be necessary